spec/actors/hyrax/actors/generic_work_actor_spec.rb

```diff
@@ -253,15 +255,17 @@
 
       before do
         curation_concern.apply_depositor_metadata(user.user_key)
-        curation_concern.save!
-        old_parent.ordered_members << curation_concern
-        old_parent.save!
+        persister.save(resource: curation_concern)
+        old_parent.member_ids << curation_concern.id
+        persister.save(resource: old_parent)
       end
+
       it "removes the old parent" do
         allow(curation_concern).to receive(:depositor).and_return(old_parent.depositor)
         expect(subject.update(env)).to be true
         expect(curation_concern.in_works).to eq []
-        expect(old_parent.reload.members).to eq []
+        reloaded = Hyrax::Queries.find_by(id: old_parent.id)
```

Comment:
How is `Hyrax::Queries.find_by` different from `Valkyrie.config.metadata_adapter.query_service.find_by`?

Reply:
I think one is just an alias to the other.

Reply:
:+1: Which do you think should win?  Should I go back and simplify the changes in my PR to use `Hyrax::Queries.find_by`?